sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonService.java

```diff
@@ -604,4 +604,17 @@ public boolean updateGeoLocation(Person person, boolean overwriteExistingCoordin
 		}
 		return geoLocationUpdated;
 	}
+
+	public List<Person> getByExternalIds(List<String> externalIds) {
```

Comment:
As `externalIds` can be of any size, a good approach is to use the `de.symeda.sormas.backend.util.IterableHelper` when using SQL `IN` statements.

See `de.symeda.sormas.backend.caze.CaseService#getByPersonUuids`